KBaseSearchEngine.spec

```diff
@@ -174,6 +174,16 @@ module KBaseSearchEngine {
         mapping<string, string> object_props;
     } ObjectData;
 
+    typedef int accessGroupId;
+    typedef string timestamp;
+
+    typedef tuple<string narrative_name,
+                  int narrative_id,
+                  timestamp time_last_saved,
+                  string ws_owner_username,
+                  string ws_owner_realname
+                  > NarrativeInfo;
```

Comment:
The standard we've been using is that tuples are `lower_case`

Reply:
Done